src/root_get.html

```diff
@@ -74,7 +80,9 @@
                 <input id="upload_submit" type="submit" class="button" value="Deploy Wasm workload" />
                 <br />
                 <br />
+                <a href="https://enarx.dev/docs/running/enarx_toml">Enarx.toml</a>:
                 <textarea class="textarea" name="toml" autofocus>
+# This configuration enables forwarding of stdin, stdout, and stderr.
```

Comment:
Great idea to add docs here. I think we should extend it with some commented-out, documented options as well, e.g. listening on sockets with tls and tcp

Reply:
see also:

https://github.com/haraldh/enarx/blob/feature/init/crates/enarx-config/src/lib.rs#L18